plugins/techdocs-backend/package.json

```diff
@@ -23,6 +23,7 @@
     "@backstage/backend-common": "^0.2.0",
     "@backstage/catalog-model": "^0.2.0",
     "@backstage/config": "^0.1.1",
+    "@backstage/core-api": "^0.2.1",
```

Comment:
i think this snuck in accidentally

Reply:
I had to add `@backstage/core-api` for the `DisoveryApi` bit. But will revert this and `yarn.lock` changes :)